### ISSUE TITLE
Changing when scrollbar is visible

### ISSUE THREAD

**Issue by arthmis** [linebender/druid#1685]
I think it would be best, in its current state, to have the scrollbar show up whenever the scroll widget is hovered over and the scroll bar will fade when the mouse leaves the scroll widget. As it is currently, you have to magically know where the scroll bar is. If you have a large list, you have to guess where it is in order to find it. At least with this new functionality, as long as you're mousing over the scroll component, the bar will be visible. I think it's an ok compromise for now between, having scroll bar visible all the time and having a scrollbar that is above everything.

**Comment by arthmis** [comment]
Could I push a PR for the change I proposed above? It's a small change. Of course there will need to be more work on the scroll component in the future.

**Comment by cmyr** [comment]
My preference would be to support an "always show scrollbars" option on the scroll view, if you're interested in implementing that?

**Comment by arthmis** [comment]
Ok that sounds good. If the view the scrollbar holds fits completely, should the scrollbar be visible still?

**Comment by arthmis** [comment]
Could there be a option to be visible on scroll hot?

**Comment by cmyr** [comment]
No, I would only show the scrollbars if there is scrollable content on that axis.

**Comment by ForLoveOfCats** [comment]
I had a WIP implementation of inlay scrollbars a while back but was unable to complete them at the time and it has since bit rotted. I highly encourage you to take a crack at this :)

**Comment by arthmis** [comment]
well, I'll take a shot at it.
